### DISCUSSION TITLE
Is self registration possible?

### DISCUSSION THREAD

**Discussion by DevKishan** [postalserver/postal#2330] [Help with using Postal]
Currently, the admin adds new users manually and assigns them root-level or organizational-level access. How can self-registration be enabled? 

If it is possible, then is there also any way through which we can set a predefined limit on the number of emails that can be sent in a month when someone creates a new account?

Thank you.

**Comment by uhlhosting** [comment]
Not from what I know. Yet you can create accounts.

**Comment by willpower232** [comment]
I don't believe either of those are features currently.
